.vscode/launch.json

```diff
@@ -148,40 +148,7 @@
             }
         },
         {
-            "name": "VS Code Tests (Jupyter+Python Extension installed, *.vscode.test.ts)",

```

Comment:
THis is the same as notebook tests, except we didn't run the notebook tests, don't see any value in that, we can use other ways to exclude tests (just a pain to keep that global env variable to exclude tests, not required as we can run those in stable & insider vscode now, when in the past we could not)